electron/walletManager.ts

```diff
@@ -168,6 +173,26 @@ export class WalletManager {
     }
   }
 
+  public clearWalletFiles(pathToClear: string) {
+    if (fs.existsSync(pathToClear)) {
+      const items = fs.readdirSync(pathToClear, { withFileTypes: true })
```

Comment:
Is it better to delete all files one by one instead of remove the `.sheikah` directory using [fs.rmdirSync](https://nodejs.org/docs/latest-v20.x/api/fs.html#fsrmdirsyncpath-options). IMO is less prone to error only try to delete once